WordPress/Sniffs/WP/I18nSniff.php

```diff
@@ -70,8 +70,14 @@ public function register() {
 	 */
 	public function process( PHP_CodeSniffer_File $phpcs_file, $stack_ptr ) {
 		$tokens = $phpcs_file->getTokens();
-		$token  = $tokens[ $stack_ptr ];
+		static $only_once = true;
+		if ( $only_once ) {
+			// echo( '$tokens = ' . var_export( $tokens, true ) . PHP_EOL ); //DEBUG
```

Comment:
The debug code here (and below on line 80) can be removed. 😃 

(And just FYI, if you have never used an IDE with support for xdebug built in, you are truly missing out! 😃)